python/mxnet/ndarray/numpy/_op.py

```diff
@@ -98,6 +99,56 @@ def ones(shape, dtype=_np.float32, order='C', ctx=None):
     return _npi.ones(shape=shape, ctx=ctx, dtype=dtype)
 
 
+@set_module('mxnet.ndarray.numpy')
+def full(shape, fill_value, dtype=None, ctx=None, out=None):
```

Comment:
Please follow the numpy's `full` API definition as the following. For now we only support `order='C'`.
```python
def full(shape, fill_value, dtype=None, order='C', ctx=None)
```

Reply:
Shall we retain the 'out' parameter for `full`? Numpy does not support 'out' but we do.